tensorflow/compiler/xla/service/tuple_simplifier.h

```diff
@@ -27,13 +27,17 @@ namespace xla {
 // the module.
 class TupleSimplifier : public HloPassInterface {
  public:
-  TupleSimplifier() {}
+  TupleSimplifier() : TupleSimplifier(false) {}
+  TupleSimplifier(bool exclude_entry_computation);
```

Comment:
Please add a brief comment explaining what is the meaning of `exclude_entry_computation` (preferably without just repeating the words in the variable name :).